{"body": "@macksmind I'm pretty sure that #7521 fixes your issue, and we haven't heard from you in months. I'm giving this a close, and if you can demonstrate that it's still a problem, we can re-open.", "url": "https://api.github.com/repos/rails/rails/issues/comments/10474453", "created_at": "2012-11-17T14:14:28Z", "html_url": "https://github.com/rails/rails/pull/6833#issuecomment-10474453", "updated_at": "2012-11-17T14:14:28Z", "repo": "rails", "issue_id": 6833, "user": {"following_url": "https://api.github.com/users/steveklabnik/following{/other_user}", "events_url": "https://api.github.com/users/steveklabnik/events{/privacy}", "organizations_url": "https://api.github.com/users/steveklabnik/orgs", "url": "https://api.github.com/users/steveklabnik", "gists_url": "https://api.github.com/users/steveklabnik/gists{/gist_id}", "html_url": "https://github.com/steveklabnik", "subscriptions_url": "https://api.github.com/users/steveklabnik/subscriptions", "avatar_url": "https://2.gravatar.com/avatar/233c279c012ebac792aaa805f966cbc7?d=https%3A%2F%2Fidenticons.github.com%2F0c22d45f31acf0f4192c8c12b7a55dee.png", "repos_url": "https://api.github.com/users/steveklabnik/repos", "received_events_url": "https://api.github.com/users/steveklabnik/received_events", "gravatar_id": "233c279c012ebac792aaa805f966cbc7", "starred_url": "https://api.github.com/users/steveklabnik/starred{/owner}{/repo}", "login": "steveklabnik", "type": "User", "id": 27786, "followers_url": "https://api.github.com/users/steveklabnik/followers"}, "owner": "rails", "_id": {"$oid": "523803b2bd3543c15100c48d"}, "id": 10474453, "issue_url": "https://api.github.com/repos/rails/rails/issues/6833"}
